constants.go

```diff
@@ -48,6 +48,13 @@ const (
       <source network='{{.Network}}'/>
       <model type='virtio'/>
     </interface>
+    <console type='pty'></console>
```

Comment:
Why not, though I'd first use `virt-viewer $vmname` for debugging VM boot issues rather than directly going through serial console. But that's probably just a matter of preference ;)